components/greaselion/browser/BUILD.gn

```diff
@@ -11,6 +11,8 @@ static_library("browser") {
     "switches.h",
   ]
 
+  configs += [ "//brave/common:updater_config" ]
```

Comment:
this is not correct, this should be automatically inherited as a public config or from build:compiler as specified below https://github.com/brave/brave-core/pull/9933/files#r700430714

Reply:
resolved!